lib/modules/platform/gerrit/scm.ts

```diff
@@ -134,10 +134,16 @@ export class GerritScm extends DefaultGitScm {
         if (commit.autoApprove) {
           pushOptions.push('label=Code-Review+2');
         }
+        if (commit.labels) {
+          for (const label of commit.labels) {
+            pushOptions.push(`hashtag=${label}`);
```

Comment:
should the values be url encoded?

Reply:
No, they should not. If you try to encode them, Gerrit will create a literal hashtag with the percentage symbols.

Also, this is the main reason why I changed from `git push origin HEAD:refs/for/master%pushOptionsHere...` to `--push-option`s. `--push-option` allow pretty much any character to be passed and will be turned into a hashtag without any escaping need.